spec.md

```diff
@@ -1359,23 +1544,15 @@ conditional statement, or as a convenient placeholder for removed components
 during transformational passes. See [@sec:conditionals] for details on the
 conditional statement.
 
-## Wires
-
-A wire is a named combinational circuit component that can be connected to and
-from using connect statements.
+## Wire
 
-The following example demonstrates instantiating a wire with the given name
-`mywire`{.firrtl} and type `UInt`{.firrtl}.
-
-``` firrtl
-wire mywire: UInt
-```
+See [@sec:wires].
 
 ## Registers
```

Comment:
If you are changing these titles to be named after the keyword, which I don't think really makes sense, shouldn't this one be called Reg?